build/configs/rtl8721csm/scripts/rlx8721d_img2_ns.ld

```diff
@@ -191,7 +192,7 @@ SECTIONS
 		*lib_vikmemcpy.o (.text*)
 		*dq_rem.o (.text*)
 		*dq_addlast.o (.text*)
-		_etext = ABSOLUTE(.);
+		_ram_kernel_text_end = ABSOLUTE(.);
```

Comment:
This change will result in some issues in our existing code. 
Where ever we are checking for stext and etext, we assume that the whole of kernel text is in flash. 
But, now if we are placing one part in flash and other in ram, then we need to change our logic in each place in our source code. Please make such required changes in src code and add it as a separate commit.